### PR TITLE
V3 "Fire in the Hole"

### PR DESCRIPTION
**Pull request by trueter**
https://github.com/membranepotential/ascstudymonitor/milestone/2

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -93,5 +93,70 @@
...
+              analytics._loadOptions = t
+            }
+            analytics.SNIPPET_VERSION = '4.1.0'
+            analytics.load(
+              window.ENV === 'production'
+                ? '8OvSbZ17xxOC5FV3RkyM0bGSPVjBCkA6'
+                : 'aN2mtGYzr15b3W6HnNLfgXIsmPnBdnwt',
+            )
+          }
+      })()
```
Comment (19h) [inline client/public/index.html:159]: please tell me this is the result of beautifying a minified tracking snippet

Comment (trueter) [reply client/public/index.html:159]: It is.

**Comment by trueter** [comment]
@19h graphql stuff getting hot here. reviews welcome :*

File: client/package.json [hunk 2]
```diff
@@ -68,6 +68,16 @@
     "parserOptions": {
       "parser": "babel-eslint"
     },
+    "rules": {
+      "no-unused-vars": [
+        "error",
+        {
+          "vars": "all",
+          "args": "after-used",
+          "ignoreRestSiblings": true
```
Comment (trueter) [inline client/package.json:73]: 👍

**Comment by trueter** [comment]
Closing to clean things up. We’re now working on the `next` branch.

**Pull request closed without merging**